src/components/Stats/styled.js

```diff
@@ -4,8 +4,8 @@ export const ArrayContainer = styled.div`
   display: flex;
   flex-direction: column-reverse;
```

Comment:
Enlève les deux column-reverse.
On va devoir remplacer l'algorythme du for.
On oublie pas la formule magique.
i < 15 - statDuPokemon

Good luck